src/locales/en_US/en_US.json

```diff
@@ -480,6 +405,16 @@
         "title": "Get your secure link",
         "url_field_label": "Copy the link to your mobile browser"
     },
+    "livenessV2": {
```

Comment:
Is this production feature? If so, it needs to be added to MIGRATION.md

Reply:
I dont know, I pulled from Lokalise as source of truth. it looks like its not used anywhere in our codebase.